samples/minimal-bundle/README.md

```diff
@@ -1,19 +1,84 @@
-# Sample
+# Sample -  Getting Started with Web Chat CDN Core Bundle
 
+## Description
 A simple web page with a maximized Web Chat and minimal additional components.
 
-This build does not contains any additional components that makes Web Chat works in a minimal way. That means, it does not contains the following components:
 
 - Adaptive Cards
 - Cognitive Services Bing Speech
 - Markdown-It
 
 # How to run
 
+- Fork and clone this repo
+- Navigate to `/Your-Local-WebChat/samples/minimal-bundle` in command line
 - Run `npx serve`
 - Browse to [http://localhost:5000/](http://localhost:5000/)
 
 # Things to try out
 
-- Type `markdown`, you should see the raw text, instead of rendered Markdown
-- Type `card weather`, you should see an error stating Adaptive Cards renderer is not found
+- Type `markdown`: you should see the raw text, instead of rendered Markdown
+- Type `card weather`: you should see an error stating Adaptive Cards renderer is not found
+- Type `hello` as a greeting: you should be able to type to the bot and receive a response in plain text
```

Comment:
This line's format is different from the previous two lines. I recommend removing the `" as a greeting"`